frontend/src/components/common/input/styles.module.scss

```diff
@@ -19,6 +19,12 @@
   resize: none;
 }
 
+.input::-webkit-outer-spin-button,
+.input::-webkit-inner-spin-button {
```

Comment:
It's not recommended to use these pseudo classes according to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-outer-spin-button)

Reply:
@fshabanov didn't found actually the other variant to hide arrows in the input type number, do you have any idea?

Reply:
It seems like there's some other solutions with `inputmode` and patterns for `text` input, but they are also not pretty common and may work unstable with different browsers
But I've found one small advice for current approach (as these pseudo classes won't work for mozilla browsers) - to add
```
.input[type='number'] {
  -moz-appearance: textfield;
}
```